commands/cli/helptext_test.go

```diff
@@ -4,7 +4,9 @@ import (
 	"strings"
 	"testing"
 
+	"bytes"
```

Comment:
The problem with making it part of the Cmd struct is that there is one command, which has two names. (i forgot which it was) If we want to keep this feature, we can not put the name in there there.